src/it/java/teammates/it/storage/sqlapi/AccountRequestsDbIT.java

```diff
@@ -192,19 +192,4 @@ public void testSqlInjectionInDeleteAccountRequest() throws Exception {
         AccountRequest actual = accountRequestDb.getAccountRequest(accountRequest.getEmail(), accountRequest.getInstitute());
         assertEquals(accountRequest, actual);
     }
-
-    @Test
-    public void testSqlInjectionSearchAccountRequestsInWholeSystem() throws Exception {
-        ______TS("SQL Injection test in searchAccountRequestsInWholeSystem");
-
-        AccountRequest accountRequest = new AccountRequest("test@gmail.com", "name", "institute");
-        accountRequestDb.createAccountRequest(accountRequest);
-
-        String searchInjection = "institute'; DROP TABLE account_requests; --";
-        List<AccountRequest> actualInjection = accountRequestDb.searchAccountRequestsInWholeSystem(searchInjection);
-        assertEquals(0, actualInjection.size());
-
-        AccountRequest actual = accountRequestDb.getAccountRequest("test@gmail.com", "institute");
-        assertEquals(accountRequest, actual);
-    }
-}
+}
```

Comment:
Let's ensure the files have trailing newlines. That means also for the `AccountRequestSearchIT.java` file.